src/components/widget/ActionButton.js

```diff
@@ -127,12 +128,16 @@ class ActionButton extends Component {
    * @todo Write the documentation
    */
   handleChangeStatus = status => {
-    const { onChange } = this.props;
+    const { onChange, docId, windowType, activeTab } = this.props;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['activeTab' is missing in props validation (react/prop-types)](https://app.codacy.com/app/metasfresh/metasfresh-webui-frontend/pullRequest?prid=4601260)